test/framework/daemonset.go

```diff
@@ -104,8 +104,8 @@ func RestartDaemonSet(t *TestArgs) {
 	t.T.Logf("%s: csi driver pods are up with no deletion timestamps", time.Now().String())
 }
 
-//TODO presumably this can go away once we have an OLM based deploy that is also integrated with our CI
-// so that repo images built from PRs are used when setting up this driver's daemonset
+//TODO possibly this can go away once we in the OCP payload, though the csi driver operator element for all that needs to get sorted out,
```

Comment:
```suggestion
//TODO possibly this can go away once we are in the OCP payload, though the csi driver operator element for all that needs to get sorted out,
```